<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetPenDialog</class>
 <widget class="QDialog" name="GetPenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GetPenDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>77</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnOk">
      <property name="text">
       <string>确定(&amp;O)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消(&amp;C)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Pen设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线  型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbStyle">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NoPen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SolidLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DashLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DotLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DashDotLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DashDotDotLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CustomDashLine</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线  宽</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>颜  色</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbWidth">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColor">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QWLabel" name="labColor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWLabel</class>
   <extends>QLabel</extends>
   <header location="global">qwlabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>GetPenDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>GetPenDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
